packages/manager/modules/netapp/src/dashboard/index/controller.js

```diff
@@ -27,6 +30,15 @@ export default class OvhManagerNetAppDashboardIndexCtrl {
     );
   }
 
+  editName() {
+    this.OvhManagerNetAppDashboardIndex.updateStorage(
+      this.storage.id,
+      this.storage.name,
+    ).then(() => {
+      window.location.reload();
```

Comment:
**nitpick:** Not the right way to reload the page. Please use `$state.go` to go to the right page.